jwst/extract_1d/extract.py

```diff
@@ -1874,59 +1917,28 @@ def extract(self, data, wl_array, verbose):
             x_array.fill((self.xstart + self.xstop) / 2.)
 
         if self.wcs is not None:
-            nelem = slice1 - slice0
-            if self.exp_type in WFSS_EXPTYPES:
-                # We expect two (x and y) or three (x, y, spectral order).
-                n_inputs = self.wcs.forward_transform.n_inputs
-                ra = np.zeros(nelem, dtype=np.float64)
-                dec = np.zeros(nelem, dtype=np.float64)
-                # Temporary variable so as not to clobber `wavelength`.
-                wcs_wl = np.zeros(nelem, dtype=np.float64)
-                transform = self.wcs.forward_transform
-                if n_inputs == 2:
-                    for i in range(nelem):
-                        ra[i], dec[i], wcs_wl[i], _ = transform(
-                                        x_array[i], y_array[i])
-                elif n_inputs == 3:
-                    for i in range(nelem):
-                        ra[i], dec[i], wcs_wl[i], _ = transform(
-                                x_array[i], y_array[i], self.spectral_order)
-                else:
-                    if verbose:
-                        log.warning("n_inputs for wcs function is %d",
-                                    n_inputs)
-                        log.warning("WCS function was expected to take "
-                                    "either 2 or 3 arguments.")
-                    ra[:] = -999.
-                    dec[:] = -999.
-                    wcs_wl[:] = -999.
-            else:
-                ra, dec, wcs_wl = self.wcs(x_array, y_array)
+            stuff = self.wcs(x_array, y_array)
```

Comment:
`stuff` seems like a really ambiguous variable name. Is there another name that might be more clear while remaining concise?